src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java

```diff
@@ -130,7 +138,15 @@ public void run() {
 
             try {
                 synchronized (this) {
-                    wait(UPDATE_DELAY);
+                    long startTime = System.nanoTime();
```

Comment:
I started with currentTimeMillis, but found several articles that recommends nanoTime instead. It should apparently handle machine time adjustments better. Like someone setting a new date/time on the machine.